app.js

```diff
@@ -34,8 +38,8 @@ const router = new Router();
 const creatorKeyJson = JSON.parse(process.env.ACCOUNT_CREATOR_KEY);
 const recoveryKeyJson = JSON.parse(process.env.ACCOUNT_RECOVERY_KEY);
 const keyStore = {
-    async getKey(networkId, accountId) {
-        if (accountId == creatorKeyJson.account_id) {
+    async getKey(accountId) {
```

Comment:
Why this change? https://github.com/nearprotocol/nearlib/blob/de49029846a95790adb81ed977a0327517148692/src.ts/key_stores/keystore.ts#L10

Reply:
I did this in its own commit, and at that time we only called `getKey` from one place in this codebase: https://github.com/nearprotocol/near-contract-helper/pull/88/commits/9805cdfb343d48c608ee8879c22446204a0bc285#diff-0364f57fbff2fabbe941ed20c328ef1aR116

```js
const helperPublicKey = (await keyStore.getKey(recoveryKeyJson.account_id)).publicKey;
```

We were only passing one argument to `getKey`, and that key appears to be an account id, not a network id. Maybe I misunderstand how this `keyStore` gets used, though?

Since that commit, I've added others which remove the call to `keyStore.getKey` altogether. Do we need to keep this `keyStore` definition around at all?

Reply:
we can remove all stuff related to `recoveryKeyJson` given we aren't doing custodial recovery anymore.

`const helperPublicKey = (await keyStore.getKey(recoveryKeyJson.account_id)).publicKey;`

this was a typo, calling `getKey` in wrong way